Source/WebCore/style/UserAgentStyle.cpp

```diff
@@ -234,8 +234,6 @@ void UserAgentStyle::ensureDefaultStyleSheetsForElement(const Element& element)
         StringBuilder fullscreenRules;
         fullscreenRules.appendCharacters(fullscreenUserAgentStyleSheet, sizeof(fullscreenUserAgentStyleSheet));
         fullscreenRules.append(RenderTheme::singleton().extraFullScreenStyleSheet());
-        if (element.document().quirks().needsBlackFullscreenBackgroundQuirk())
-            fullscreenRules.append(":-webkit-full-screen { background-color: black; }"_s);
         fullscreenStyleSheet = parseUASheet(fullscreenRules.toString());
         addToDefaultStyle(*fullscreenStyleSheet);
```

Comment:
Since `extraFullscreenStyleSheet()` is always an empty string (see: https://searchfox.org/wubkat/search?q=extraFullScreenStyleSheet&path=&case=false&regexp=false), we can simplify this and this can become:
```
fullscreenStyleSheet = parseUASheet(StringImpl::createWithoutCopying(fullscreenUserAgentStyleSheet, sizeof(fullscreenUserAgentStyleSheet)));
addToDefaultStyle(*fullscreenStyleSheet);
```

Reply:
I will do a cycle of recompile. Might take a bit of time.

Reply:
ok this is working locally in my own build.

Reply:
OK repushing.
Modified the line as you mentioned. Thanks.

Reply:
5th time the charm 😄 pushed.